packages/devtools_app/lib/src/screens/memory/panes/diff/widgets/snapshot_control_pane.dart

```diff
@@ -46,18 +47,23 @@ class SnapshotControlPane extends StatelessWidget {
                 ],
               ],
             ),
-            ToolbarAction(
-              icon: Icons.clear,
-              tooltip: 'Delete snapshot',
-              onPressed: isProcessing
-                  ? null
-                  : () {
-                      controller.deleteCurrentSnapshot();
-                      ga.select(
-                        gac.memory,
-                        gac.MemoryEvent.diffSnapshotDelete,
-                      );
-                    },
+            Expanded(
+              child: Row(
+                mainAxisAlignment: MainAxisAlignment.end,
+                children: [
+                  if (heapIsReady) ...[
+                    Expanded(
+                      child:
+                          _SnapshotSizeView(footprint: current.heap!.footprint),
```

Comment:
Ubernit: adding a trailing comma after `footprint` will make this format more cleanly.